src/main/assets/scss/main.scss

```diff
@@ -123,3 +123,15 @@ tbody {
 .no_padding {
   padding: 0;
 }
+
+.table-column-court-number {
+  width: 12%;
+}
+
+.table-header-bottom-align {
+  vertical-align: bottom;
+}
+
+.no-information-message {
```

Comment:
No worries if not, however is there any way we can use the gov margin override classes for this instead